server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -301,6 +310,11 @@ def get_sql(self, **kwargs: Any) -> str:
             query=step.query.replace("##PREVIOUS_STEP##", table.name),
         )
 
+        custom_query.query = (
+            custom_query.query.replace(";", "")
```

Comment:
why is this needed ?

Reply:
a missed !
I was to removed the ";" for custom queries, since, none-pypika fixture fixture was failing on it : https://github.com/ToucanToco/weaverbird/blob/master/server/tests/backends/fixtures/customsql/simple.json#L17

I just removed that : https://github.com/ToucanToco/weaverbird/pull/1390/commits/ea7945089bb792e44353dbf227340a2d770526a3